tommath.h

```diff
@@ -167,9 +168,10 @@ typedef int mp_order;
 #define MP_LSB_FIRST -1
 #define MP_MSB_FIRST  1
 typedef int mp_endian;
-#define MP_LTTLE_ENDIAN  -1
+#define MP_LITTLE_ENDIAN  -1
 #define MP_NATIVE_ENDIAN  0
 #define MP_BIG_ENDIAN     1
+#define MP_UNKNOWN_ENDIAN 1
```

Comment:
Please no unknown endian!

Reply:
The rest of LTM is endianness agnostic.
I think?

Reply:
Yes but we shouldn't added this to the mp_,endian enum values.

Reply:
Ping @czurnieden - could you remove unknown_endian everywhere?